docs/en/connector-v2/sink/Easysearch.md

```diff
@@ -0,0 +1,178 @@
+# INFINI Easysearch
```

Comment:
Please refer to the new document style

https://github.com/apache/seatunnel/blob/dev/docs/en/connector-v2/sink/IoTDB.md